test/unit-test/depacketization/depacketization_utest.c

```diff
@@ -157,6 +162,278 @@ void test_G711_Depacketizer( void )
     TEST_ASSERT_EQUAL_UINT8_ARRAY( &( decodedFrame[ 0 ] ),
                                    &( frame.pFrameData[ 0 ] ),
                                    frame.frameDataLength );
+    pkt.pPacketData = &( packetData1[ 0 ] );
+    pkt.packetDataLength = sizeof( packetData1 );
+    result = G711Depacketizer_GetPacketProperties( pkt.pPacketData,
```

Comment:
Why is this G711Depacketizer_GetPacketProperties needed here? Does packetData1 have any pre-config property?
- If it has pre-config property, please add a check on returned properties.
- If no, it's not necessary to add it here.

Reply:
Right, I will make a separate unit-test for that